### DISCUSSION TITLE
Compatibility with Nifi 1.24.0

### DISCUSSION THREAD

**Discussion by oyeyemi** [easy-global-market/nifi-file-identity-provider-bundle#32] [Ideas]
I really appreciate your effort to bring this useful feature back to Nifi.
Please, for a newbie like me, i would like you to help us with a step-by-step tutorials on how to implement this on Apache Nifi 1.24.0 and beyond.
Thank you very much.

**Comment by bobeal** [comment]
Hi @oyeyemi,

Sorry, never saw this discussion started 10 months ago...

We are using this module with NiFi 1.27.0, so it should also work with NiFi 1.24.0. Let me know if you have any problem using it.

Benoit.
